src/main/java/com/buyanywhere/productcatalog/controllers/v1/CategoryController.java

```diff
@@ -1 +1,33 @@
+package com.buyanywhere.productcatalog.controllers.v1;
 
+import com.buyanywhere.productcatalog.exceptions.CategoryNotFoundException;
+import org.springframework.web.bind.annotation.*;
+import com.buyanywhere.productcatalog.models.Category;
+import com.buyanywhere.productcatalog.repositories.CategoryRepository;
+
+@RestController
+@RequestMapping("/v1/category")
+
+public class CategoryController {
+
+    private CategoryRepository repository;
+
+    public CategoryController(CategoryRepository repository) {
+        this.repository = repository;
+    }
+
+    @RequestMapping(method = RequestMethod.POST, value = "/create-new")
```

Comment:
In the previous PR for category creation you have the correct RequestMapping, you don't use the create-new since this is REST Api, so the methods names and Urls should be depending on the verbs only when possible (get, post, put, delete)